packages/nitro-protocol/package.json

```diff
@@ -11,42 +11,26 @@
   },
   "devDependencies": {
     "@statechannels/devtools": "0.1.1",
-    "@statechannels/jest-gas-reporter": "0.0.2",
     "@types/eslint": "6.1.7",
     "@types/eslint-plugin-prettier": "2.2.0",
     "@types/jest": "25.1.0",
-    "@types/koa": "2.0.52",
-    "@types/koa-router": "7.0.42",
     "@types/node": "13.5.1",
     "@types/prettier": "1.19.0",
-    "@types/server-destroy": "1.0.0",
+    "@types/webpack": "4.41.5",
     "@typescript-eslint/eslint-plugin": "2.18.0",
     "@typescript-eslint/parser": "2.18.0",
-    "dotenv": "8.2.0",
-    "dotenv-extended": "2.7.1",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.10.0",
-    "eslint-plugin-import": "2.20.0",
     "eslint-plugin-jest": "23.6.0",
     "eslint-plugin-prettier": "3.1.2",
-    "etherlime": "2.3.3",
-    "etherlime-lib": "1.1.5",
-    "ganache-cli": "6.8.2",
-    "ganache-core": "2.8.0",
```

Comment:
Removing this will prevent us from needlessly trying and failing to build scrypt on every `yarn install`:
```
➜  monorepo git: yarn why scrypt
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "scrypt"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "scrypt@6.0.3"
info Reasons this module exists
   - "_project_#@statechannels#nitro-protocol#ganache-core#ethereumjs-wallet#scrypt.js" depends on it
   - Hoisted from "_project_#@statechannels#nitro-protocol#ganache-core#ethereumjs-wallet#scrypt.js#scrypt"
✨  Done in 4.48s.
```

I am discovering that it is even worse on a `yarn add` than it is on a `yarn install`, because errors in building `scrypt` cause the whole install to fail. 

So this is a pretty criticial improvement IMO.

Reply:
Distilled a more minimal version of this change into #2268.